scripted-tests/run/java-io-file/Files.scala

```diff
@@ -0,0 +1,112 @@
+import java.io.File
+
+object Files {
+
+  val emptyNameFile = new File("")
+
+  val executableFile        = new File("executableFile.txt")
+  val unexecutableFile      = new File("unexecutableFile.txt")
+  val executableDirectory   = new File("executableDirectory")
+  val unexecutableDirectory = new File("unexecutableDirectory")
+
+  val readableFile        = new File("readableFile.txt")
+  val unreadableFile      = new File("unreadableFile.txt")
+  val readableDirectory   = new File("readableDirectory")
+  val unreadableDirectory = new File("readableleDirectory")
+
+  val writableFile        = new File("writableFile.txt")
+  val unwritableFile      = new File("unwritableFile.txt")
+  val writableDirectory   = new File("writableDirectory")
+  val unwritableDirectory = new File("unwritableDirectory")
+
+  // Used to test `File.compareTo`
+  val fileA = new File("A")
+  val fileB = new File("B")
+
+  val willBeCreatedFile                = new File("willBeCreatedFile.txt")
+  val willBeCreatedDirectory           = new File("willBeCreatedDirectory")
+  val nestedWillBeCreatedDirectoryBase = new File("d0")
+  val nestedWillBeCreatedDirectory = new File(
+    nestedWillBeCreatedDirectoryBase,
+    "d1/nestedwillBeCreatedDirectory")
+  val willBeDeletedFile      = new File("willBeDeletedFile.txt")
+  val willBeDeletedDirectory = new File("willBeDeletedDirectory")
+
+  val nonexistentFile      = new File("nonexistentFile.txt")
+  val nonexistentDirectory = new File("nonexistentDirectory")
+
+  // TODO: This will stop working once we support Windows
```

Comment:
Please mention this on the windows-related issue discussion, instead of leaving comment in the code. Comments are getting outdated too quickly, and typically noone is going to see them until it's too late.